server/server/src/billing/stripe_model.rs

```diff
@@ -0,0 +1,72 @@
+use serde::{Deserialize, Serialize};
+
+// Since certain fields can either be a String/Int or a Struct, I made this to handle either situation
```

Comment:
Explain?

Reply:
I updated the comment to be more comprehensive.